pkg/kubectl/service.go

```diff
@@ -57,7 +57,7 @@ func paramNames() []GeneratorParam {
 		// port will be used if a user specifies --port OR the exposed object
 		// has one port
 		{"port", false},
-		// ports will be used iff a user doesn't specify --port AND the
+		// ports will be used if a user doesn't specify --port AND the
```

Comment:
https://en.wikipedia.org/wiki/Iff

It is a technical term that anyone in computer science should understand. I am not sure if there is a big advantage to using the long form, since it is widely used and imho easier to read.